src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -374,6 +376,7 @@ void CreateInteractionChoiceSetRequest::onTimeOut() {
   if (!error_from_hmi_) {
     SendResponse(false, mobile_apis::Result::GENERIC_ERROR);
   }
+  CommandRequestImpl::onTimeOut();
```

Comment:
@JenkinsSDLOnCloud looks like needed. CommandRequestImpl::onTimeOut() will return as completed request.  I didn't investigate it deeply. But checked ATF